### PR TITLE
Release v0.11.0

### PR DESCRIPTION
**Pull request by ascjones**
### Added
- Add type parameter getters [(#122)](https://github.com/paritytech/scale-info/pull/122)
- Add support for Range and RangeInclusive [(#124)](https://github.com/paritytech/scale-info/pull/124), [(#126)](https://github.com/paritytech/scale-info/pull/126)
- Explicit codec indices for `TypeDef` and `TypeDefPrimitive` enums [(#127)](https://github.com/paritytech/scale-info/pull/127)

Also this moves the `Range` variant to the end, for backwards compatibility. /cc @jacogr

### CODE REVIEW

File: src/ty/mod.rs [hunk 1]
```diff
@@ -247,14 +247,14 @@ pub enum TypeDef<T: Form = MetaForm> {
...
-    /// A Range type.
-    Range(TypeDefRange<T>),
     /// A Rust primitive type.
     Primitive(TypeDefPrimitive),
     /// A type using the [`Compact`] encoding
     Compact(TypeDefCompact<T>),
     /// A type representing a sequence of bits.
     BitSequence(TypeDefBitSequence<T>),
+    /// A Range type.
+    Range(TypeDefRange<T>),
```
Comment (dvdplm) [inline src/ty/mod.rs:257]: Oh good point. Do you mind mentioning this in the docs as well?

Comment (ascjones) [reply src/ty/mod.rs:257]: That it is backwards compatible, or that new variants should be added to the end?

Comment (dvdplm) [reply src/ty/mod.rs:257]: The latter. Something like "Tooling relying on scale-info data generally relies on order to find items. For this reason it is important that public enums add new variants items to the end to ensure backwards compatibility"?

Comment (jacogr) [reply src/ty/mod.rs:257]: +1

Comment (ascjones) [reply src/ty/mod.rs:276]: I've gone a step further and added explicit indices for this and `TypeDefPrimitive`, in case people delete or reorder variants.

**Review by dvdplm — APPROVED** [review]

**Pull request merged**